src/components/Gallery/index.js

```diff
@@ -11,10 +11,10 @@ const Gallery = () => {
                 <h1>{gallery.title}</h1>
                 <h3>{gallery.subTitle}</h3>
             </div>
-            
+
             <div className="gallery__images">
                 {gallery.galleryImages.map((gallery, index) => {
-                    return ( <img src={gallery['path']} key ={index} /> );
+                    return ( <img src = {gallery['path']} key = {index} /> );
```

Comment:
Sorry for the confusion, need to remove the space `src = {gallery['path']}` `key = {index}`. Since in other components you haven't space between `=` and `{`
`( <img src={gallery['path']} key={index} /> );`